### PR TITLE
Modified installation instructions to reflect switch from Astroconda …

### PR DESCRIPTION
**Pull request by shahm0**
…to conda-forge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,11 +19,8 @@ The simplest way to get the most recent environment would be:
 2.  Create a conda environment with the HSTCAL included:
 
 ```bash
-# Add the AstroConda channel to ~/.condarc
-$ conda channel --add http://ssb.stsci.edu/astroconda
-
 # Create a dedicated calibration environment
-$ conda create -n calib python=3 stsci-hst
+$ conda create -n calib --file [insert URL of conda environment yaml file]
```
Comment (pllim) [inline README.md:23]: FWIW, this is not necessarily related to conda-forge. It just asking user to provide a YAML file.

**Comment by shahm0** [comment]
Changed to a draft PR since we don't want to publish these new instructions until a new build is available.

**Comment by mdlpstsci** [comment]
@nden @zacharyburnett If you are satisfied, then the installation instructions should be updated.  I assume you just want folks to download the YML file from https://github.com/spacetelescope/stenv/releases 
and then generate the Conda environment?

FYI - We are on RC1 for a new build.  While a new version of HSTCAL is not currently part of the build, this is a minor change.

**Comment by zacharyburnett** [comment]
> @nden @zacharyburnett If you are satisfied, then the installation instructions should be updated. I assume you just want folks to download the YML file from https://github.com/spacetelescope/stenv/releases and then generate the Conda environment?

Yes, I think that would be ideal; the `stenv` documentation should helpful for users to spin up an environment using `conda-forge`. If the YAML files in the latest release don't work, users can also fallback to the generalized YAML files in the root of `stenv` (this is also covered in the docs).

**Comment by mdlpstsci** [comment]
@shahm0 Please proceed with this ticket as a build involving updates to HSTCAL and Drizzlepac are being formulated.

**Comment by mdlpstsci** [comment]
This PR has been resolved by PR#589.

**Pull request closed without merging**